src/app/(main)/EventPage.tsx

```diff
@@ -51,8 +52,20 @@ const Roster = ({participants, orgs, startTime}: {participants: Record<string, P
       const isToday = new Date().setHours(0,0,0,0) === new Date(o.value).setHours(0,0,0,0);
       return `${!isToday ? formatDate(o.value, 'yyyy-MM-dd ') : ''}${formatDate(o.value, 'HHmm')}`;
     }, flex: 1 },
+    { field: 'eta', headerName: 'ETA', flex: 1},
   ];
 
+  function CustomToolbar() {
+    return (
+      <GridToolbarContainer>
+        <GridToolbarColumnsButton />
+        {false && <GridToolbarFilterButton />}

```

Comment:
What does this line do, and what's the "false &&" for?

Reply:
It's a shorthand for "if this is true, then render that". Normally there's a boolean evaluation in the first part, but I just put false to hide that element for now.